### ISSUE TITLE
Talk proposal: <talk title>

### ISSUE THREAD

**Issue by HelloDeadline** [queerjs/queerjs-talk-proposals#4]
### Talk title

Let me illustrate it for you!

### Abstract 

What do you do when your first ever conference talk gets accepted? Panic! Experiment! Prepare for a major failure! What if it was possible to make it even riskier and involve someone else in this adventure? Someone with a completely different mindset, someone with the skills you lack? An illustrator!

This is a story about collaboration between developer and illustrator. It shows that diversity and trust help to achieve the most ambitious and scariest goals. And improve lives of everyone involved in a surprisingly unpredictable way! So how to experience diversity even if your team at work is not that diverse? How to get the most of it? How to organize the creative process, stick to the goal and give constructive feedback? How to stay excited and enjoy the adventure? Let's find out!

### Your name

Svetlana Sharipova

### Location

Based in: Saint-Petersburg, Russia
Want to speek in: Stockholm

 ### Contact Details

Twitter (link it): [@hellodeadline](https://twitter.com/hellodeadline)
Email: hellodeadline.sn@gmail.com

 ### Code of Conduct

 - [x] I've read and agree to the [QueerJS Code of Conduct](https://queerjs.com/code-of-conduct)

**Comment by carolstran** [comment]
Hey hey @HelloDeadline, so nice to see you here! Loved your talk on DACHFest 💓 

I updated the issue title to have the talk title and added the `Stockholm` label since you'd like to speak there. Would you need travel assistance to get to Stockholm?

**Comment by HelloDeadline** [comment]
Oh, the title. Sorry for that and thanks for the update.

Travel assistance would be perfect. Let me know if it's possible with the meetup budget. (If not, I'll figure it out somehow.)

**Comment by carolstran** [comment]
Literally no problem at all, I was just letting you know so you weren't like wtf why did she change my title 😂 But cool, thanks for letting us know! One of the organizers who will be in Stockholm will reach out again soon 😊

**Comment by HelloDeadline** [comment]
The talk is submitted as lightning but I tested it ⏲and figured out I have enough content for a Feature Talk (20-25 minutes). Both formats work great for me. So if it's accepted, just let me know which length is better for the program.

**Comment by SaraVieira** [comment]
I spoke to her about the visa and all good, adding you now!

THANK YOU! BRING SOCKS!

**Comment by SaraVieira** [comment]
🎉 🎉 https://queerjs.com/stockholm/  🎉 🎉

Are you okay with being announced?

**Comment by HelloDeadline** [comment]
> 🎉 🎉 https://queerjs.com/stockholm/ 🎉 🎉
> 
> Are you okay with being announced?

Hell yes I am! Thank you so much!

**Comment by HelloDeadline** [comment]
Unfortunately, I won't be able to give this talk in Stockholm. The visa application process was changed and there is a queue wich makes it impossible to get it earlier than 8 Oct. I wish I could teleport as soon as they give me my passport back.

Sorry for not notifying earlier, I wish I knew earlier myself. I did what I could. I hope it opens an opportunity for someone else.

❤

**Comment by carolstran** [comment]
Hey hey @HelloDeadline, sorry I answered you on Twitter but not here 🙈 But I'll say the same here: We totally understand and please don't worry about the notice. We really, really hope to have you speak at a QueerJS soon 💜 

Would you like us to close this issue or keep it open/on the waitlist for a future event?

**Comment by HelloDeadline** [comment]
I'd like to give this talk at future event. So let's keep it open. Thank you <3

**Comment by HelloDeadline** [comment]
Are there any slots available for my talk at QueerJS in Berlin?

**Comment by carolstran** [comment]
Yes! It'd be great to have you at the Berlin event 😊  Should we add you to the website right away or wait until the visa is sorted?

**Comment by HelloDeadline** [comment]
I have an open visa, so I'll be there for sure! You can add me on a website ✨

**Comment by SaraVieira** [comment]
Yasss!!!


About travelling, we will get you a flight and some accommodation, when do you wanna fly?

**Comment by SaraVieira** [comment]
I will also add you to the website today :)

**Comment by HelloDeadline** [comment]
> Yasss!!!
> 
> About travelling, we will get you a flight and some accommodation, when do you wanna fly?

Oh, wow, this is so awesome! I was thinking 25 - 29, it would be nice to spend couple of days in Berlin after the meetup. Btw, I'm flying from Saint Petersburg.

**Comment by SaraVieira** [comment]
Hey!

So we have a small issue with the flights and by that i mean wtf why are they so expensive?

we can get you the flights but we do have a max of 300 and the flights are 218$, since all the ones with layovers have like 9h layovers :/

You can stay in my house with my wife and that's okay by us please let us know so we can get the flights and figure it out

**Comment by HelloDeadline** [comment]
> Hey!
> 
> So we have a small issue with the flights and by that i mean wtf why are they so expensive?
> 
> we can get you the flights but we do have a max of 300 and the flights are 218$, since all the ones with layovers have like 9h layovers :/
> 
> You can stay in my house with my wife and that's okay by us please let us know so we can get the flights and figure it out

It is absolutely fine if you cover only flights! 
As for staying in your house, thank you so much for offering, it would be great!

**Comment by SaraVieira** [comment]
Awesome! Can you please email me at hey@iamsaravieira.com your passport details ?
